### ISSUE TITLE
Segmentation violation on repeat usage of `podman search ...`

### ISSUE THREAD

**Issue by TuckerVaughan** [containers/podman#21696]
### Issue Description

Every time I try to run podman search on any external image, regardless of the image for the docker container in which I am running podman, it seems to run into a segmentation violation on the second or greater call. Echo and other available commands in the container still work, but podman no longer does.

### Steps to reproduce the issue

Steps to reproduce the issue
1. Spin up a docker container using any image. I used `docker run -it --rm quay.io/podman/stable /bin/sh` in case you find an image that doesn't have this issue.
2. Next, run `podman search --format json --list-tags mcr.microsoft.com/dotnet/sdk`. This instance should return the expected results and work nominally.
3. Finally, run `podman search --format json --list-tags mcr.microsoft.com/dotnet/sdk` again. This one will encounter an error and fail. The error message should start with:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x559bcfd2e1a9]

...
```

You should see that commands such as `echo` still work normally, but any further attempt to call `podman search` will fail with the same error.

### Describe the results you received

```
PS C:\...\...> docker run -it --rm quay.io/podman/stable /bin/sh
sh-5.2# podman search --format json --list-tags mcr.microsoft.com/dotnet/sdk
[
    {
        "Name": "mcr.microsoft.com/dotnet/sdk",
        "Tags": [
            "2.1",
            "2.1-alpine",
            "2.1-alpine3.12",
            "2.1-alpine3.13",
            "2.1-bionic",
            "2.1-bionic-arm32v7",
            "2.1-focal",
            "2.1-focal-arm32v7",
            "2.1-nanoserver-1809",
            "2.1-nanoserver-1903",
            "2.1-nanoserver-1909",
            "2.1-nanoserver-2004",
            "2.1-nanoserver-2009",
            "2.1-nanoserver-20H2",
            "2.1-stretch",
            "2.1-stretch-arm32v7",
            "2.1.811",
            "2.1.811-alpine3.12",
            "2.1.811-bionic",
            "2.1.811-bionic-arm32v7",
            "2.1.811-focal",
            "2.1.811-focal-arm32v7",
            "2.1.811-nanoserver-1809",
            "2.1.811-nanoserver-1903",
            "2.1.811-nanoserver-1909"
        ]
    }
]
sh-5.2# podman search --format json --list-tags mcr.microsoft.com/dotnet/sdk
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x559bcfd2e1a9]

goroutine 50 [running]:
panic({0x559bd0989da0?, 0x559bd173e8a0?})
        /usr/lib/golang/src/runtime/panic.go:1017 +0x3ac fp=0xc000082c40 sp=0xc000082b90 pc=0x559bcf190d8c
runtime.panicmem(...)
        /usr/lib/golang/src/runtime/panic.go:261
runtime.sigpanic()
        /usr/lib/golang/src/runtime/signal_unix.go:861 +0x378 fp=0xc000082ca0 sp=0xc000082c40 pc=0x559bcf1a8778
github.com/containers/common/libimage.(*Runtime).systemContextCopy(0x0)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/containers/common/libimage/runtime.go:87 +0x29 fp=0xc000082d10 sp=0xc000082ca0 pc=0x559bcfd2e1a9
github.com/containers/common/libimage.(*Runtime).searchImageInRegistry(0x0?, {0x559bd0bdd778, 0xc0004222a0}, {0x7ffc24df5f05, 0xa}, {0x7ffc24df5ef3, 0x11}, 0xc000429180)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/containers/common/libimage/search.go:165 +0x66 fp=0xc000082f10 sp=0xc000082d10 pc=0x559bcfd344e6
github.com/containers/common/libimage.(*Runtime).Search.func1()
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/containers/common/libimage/search.go:134 +0xfb fp=0xc000082fe0 sp=0xc000082f10 pc=0x559bcfd3429b
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000082fe8 sp=0xc000082fe0 pc=0x559bcf1c7f21
created by github.com/containers/common/libimage.(*Runtime).Search in goroutine 1
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/containers/common/libimage/search.go:131 +0x3f4

goroutine 1 [runnable]:
runtime.gopark(0x7fda65171fc8?, 0x7fda65166108?, 0x0?, 0xe0?, 0xc00050ec00?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0005b7618 sp=0xc0005b75f8 pc=0x559bcf1940ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.semacquire1(0xc000617378, 0x70?, 0x1, 0x0, 0x2b?)
        /usr/lib/golang/src/runtime/sema.go:160 +0x21c fp=0xc0005b7680 sp=0xc0005b7618 pc=0x559bcf1a591c
sync.runtime_Semacquire(0xc0004e45b0?)
        /usr/lib/golang/src/runtime/sema.go:62 +0x25 fp=0xc0005b76b8 sp=0xc0005b7680 pc=0x559bcf1c3c65
sync.(*WaitGroup).Wait(0xc0004f6280?)
        /usr/lib/golang/src/sync/waitgroup.go:116 +0x48 fp=0xc0005b76e0 sp=0xc0005b76b8 pc=0x559bcf1d4b48
github.com/containers/common/libimage.(*Runtime).Search(0x0, {0x559bd0bdd778?, 0xc0004222a0}, {0x7ffc24df5ef3, 0x1c}, 0xc000429180)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/containers/common/libimage/search.go:139 +0x5a8 fp=0xc0005b7820 sp=0xc0005b76e0 pc=0x559bcfd33f48
github.com/containers/podman/v4/pkg/domain/infra/abi.(*ImageEngine).Search(0xc00066e360, {0x559bd0bdd778, 0xc0004222a0}, {0x7ffc24df5ef3, 0x1c}, {{0x0, 0x0}, {0x0, 0x0}, {0x0, ...}, ...})
        /builddir/build/BUILD/podman-4.9.0/pkg/domain/infra/abi/images.go:498 +0x1c5 fp=0xc0005b7878 sp=0xc0005b7820 pc=0x559bd01599a5
github.com/containers/podman/v4/cmd/podman/images.imageSearch(0x559bd1765580?, {0xc000624d00?, 0x1?, 0x4?})
        /builddir/build/BUILD/podman-4.9.0/cmd/podman/images/search.go:156 +0x253 fp=0xc0005b7ba8 sp=0xc0005b7878 pc=0x559bd026a3b3
github.com/spf13/cobra.(*Command).execute(0x559bd1765580, {0xc000040200, 0x4, 0x4})
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:983 +0xabc fp=0xc0005b7d48 sp=0xc0005b7ba8 pc=0x559bcf6eaebc
github.com/spf13/cobra.(*Command).ExecuteC(0x559bd175e560)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:1115 +0x3ff fp=0xc0005b7e20 sp=0xc0005b7d48 pc=0x559bcf6eb77f
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:1032
main.Execute()
        /builddir/build/BUILD/podman-4.9.0/cmd/podman/root.go:115 +0xb8 fp=0xc0005b7ea8 sp=0xc0005b7e20 pc=0x559bd038e3f8
main.main()
        /builddir/build/BUILD/podman-4.9.0/cmd/podman/main.go:60 +0x467 fp=0xc0005b7f40 sp=0xc0005b7ea8 pc=0x559bd038db27
runtime.main()
        /usr/lib/golang/src/runtime/proc.go:267 +0x2d2 fp=0xc0005b7fe0 sp=0xc0005b7f40 pc=0x559bcf193c52
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005b7fe8 sp=0xc0005b7fe0 pc=0x559bcf1c7f21

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006cfa8 sp=0xc00006cf88 pc=0x559bcf1940ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.forcegchelper()
        /usr/lib/golang/src/runtime/proc.go:322 +0xb8 fp=0xc00006cfe0 sp=0xc00006cfa8 pc=0x559bcf193f38
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006cfe8 sp=0xc00006cfe0 pc=0x559bcf1c7f21
created by runtime.init.7 in goroutine 1
        /usr/lib/golang/src/runtime/proc.go:310 +0x1a

goroutine 18 [GC sweep wait]:
runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000068778 sp=0xc000068758 pc=0x559bcf1940ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.bgsweep(0x0?)
        /usr/lib/golang/src/runtime/mgcsweep.go:321 +0xdf fp=0xc0000687c8 sp=0xc000068778 pc=0x559bcf17e53f
runtime.gcenable.func1()
        /usr/lib/golang/src/runtime/mgc.go:200 +0x25 fp=0xc0000687e0 sp=0xc0000687c8 pc=0x559bcf173665
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000687e8 sp=0xc0000687e0 pc=0x559bcf1c7f21
created by runtime.gcenable in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:200 +0x66

goroutine 19 [GC scavenge wait]:
runtime.gopark(0x86a3c?, 0x3b9aca00?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000068f70 sp=0xc000068f50 pc=0x559bcf1940ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.(*scavengerState).park(0x559bd183db80)
        /usr/lib/golang/src/runtime/mgcscavenge.go:425 +0x49 fp=0xc000068fa0 sp=0xc000068f70 pc=0x559bcf17bdc9
runtime.bgscavenge(0x0?)
        /usr/lib/golang/src/runtime/mgcscavenge.go:658 +0x59 fp=0xc000068fc8 sp=0xc000068fa0 pc=0x559bcf17c379
runtime.gcenable.func2()
        /usr/lib/golang/src/runtime/mgc.go:201 +0x25 fp=0xc000068fe0 sp=0xc000068fc8 pc=0x559bcf173605
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000068fe8 sp=0xc000068fe0 pc=0x559bcf1c7f21
created by runtime.gcenable in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:201 +0xa5

goroutine 3 [finalizer wait]:
runtime.gopark(0x198?, 0x559bd0badc20?, 0x1?, 0x52?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006c620 sp=0xc00006c600 pc=0x559bcf1940ce
runtime.runfinq()
        /usr/lib/golang/src/runtime/mfinal.go:193 +0x107 fp=0xc00006c7e0 sp=0xc00006c620 pc=0x559bcf172687
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006c7e8 sp=0xc00006c7e0 pc=0x559bcf1c7f21
created by runtime.createfing in goroutine 1
        /usr/lib/golang/src/runtime/mfinal.go:163 +0x3d

goroutine 4 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006d750 sp=0xc00006d730 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc00006d7e0 sp=0xc00006d750 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006d7e8 sp=0xc00006d7e0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 20 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000069750 sp=0xc000069730 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0000697e0 sp=0xc000069750 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000697e8 sp=0xc0000697e0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 34 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000508750 sp=0xc000508730 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0005087e0 sp=0xc000508750 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005087e8 sp=0xc0005087e0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 5 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006df50 sp=0xc00006df30 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc00006dfe0 sp=0xc00006df50 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006dfe8 sp=0xc00006dfe0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 21 [GC worker (idle)]:
runtime.gopark(0x5ed7ab327a8?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000069f50 sp=0xc000069f30 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc000069fe0 sp=0xc000069f50 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000069fe8 sp=0xc000069fe0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 35 [GC worker (idle)]:
runtime.gopark(0x5ed7a705836?, 0x3?, 0x24?, 0x97?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000508f50 sp=0xc000508f30 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc000508fe0 sp=0xc000508f50 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000508fe8 sp=0xc000508fe0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 6 [GC worker (idle)]:
runtime.gopark(0x5ed7ab2ff6f?, 0x3?, 0x20?, 0x3?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006e750 sp=0xc00006e730 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc00006e7e0 sp=0xc00006e750 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006e7e8 sp=0xc00006e7e0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 22 [GC worker (idle)]:
runtime.gopark(0x559bd18774c0?, 0x1?, 0x36?, 0x52?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006a750 sp=0xc00006a730 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc00006a7e0 sp=0xc00006a750 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006a7e8 sp=0xc00006a7e0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 36 [GC worker (idle)]:
runtime.gopark(0x5ed7ab3203d?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000509750 sp=0xc000509730 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0005097e0 sp=0xc000509750 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005097e8 sp=0xc0005097e0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 7 [GC worker (idle)]:
runtime.gopark(0x5ed7ac7b18a?, 0x1?, 0x4?, 0xe7?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006ef50 sp=0xc00006ef30 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc00006efe0 sp=0xc00006ef50 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006efe8 sp=0xc00006efe0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 23 [GC worker (idle)]:
runtime.gopark(0x559bd18774c0?, 0x1?, 0xb5?, 0x7f?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006af50 sp=0xc00006af30 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc00006afe0 sp=0xc00006af50 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006afe8 sp=0xc00006afe0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 37 [GC worker (idle)]:
runtime.gopark(0x5ed7ab328d4?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000509f50 sp=0xc000509f30 pc=0x559bcf1940ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc000509fe0 sp=0xc000509f50 pc=0x559bcf175225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000509fe8 sp=0xc000509fe0 pc=0x559bcf1c7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 24 [select, locked to thread]:
runtime.gopark(0xc00050a7a8?, 0x2?, 0x0?, 0x0?, 0xc00050a7a4?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00050a638 sp=0xc00050a618 pc=0x559bcf1940ce
runtime.selectgo(0xc00050a7a8, 0xc00050a7a0, 0x0?, 0x0, 0x0?, 0x1)
        /usr/lib/golang/src/runtime/select.go:327 +0x725 fp=0xc00050a758 sp=0xc00050a638 pc=0x559bcf1a48e5
runtime.ensureSigM.func1()
        /usr/lib/golang/src/runtime/signal_unix.go:1014 +0x1a5 fp=0xc00050a7e0 sp=0xc00050a758 pc=0x559bcf1be525
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00050a7e8 sp=0xc00050a7e0 pc=0x559bcf1c7f21
created by runtime.ensureSigM in goroutine 1
        /usr/lib/golang/src/runtime/signal_unix.go:997 +0xc8

goroutine 8 [syscall]:
runtime.notetsleepg(0x0?, 0x0?)
        /usr/lib/golang/src/runtime/lock_futex.go:236 +0x29 fp=0xc0005047a0 sp=0xc000504768 pc=0x559bcf165429
os/signal.signal_recv()
        /usr/lib/golang/src/runtime/sigqueue.go:152 +0x29 fp=0xc0005047c0 sp=0xc0005047a0 pc=0x559bcf1c43c9
os/signal.loop()
        /usr/lib/golang/src/os/signal/signal_unix.go:23 +0x13 fp=0xc0005047e0 sp=0xc0005047c0 pc=0x559bcf29e773
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005047e8 sp=0xc0005047e0 pc=0x559bcf1c7f21
created by os/signal.Notify.func1.1 in goroutine 1
        /usr/lib/golang/src/os/signal/signal.go:151 +0x1f

goroutine 9 [select]:
runtime.gopark(0xc000504fb0?, 0x2?, 0x0?, 0x0?, 0xc000504eac?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000083d38 sp=0xc000083d18 pc=0x559bcf1940ce
runtime.selectgo(0xc000083fb0, 0xc000504ea8, 0x0?, 0x0, 0x0?, 0x1)
        /usr/lib/golang/src/runtime/select.go:327 +0x725 fp=0xc000083e58 sp=0xc000083d38 pc=0x559bcf1a48e5
github.com/containers/podman/v4/libpod/shutdown.Start.func1()
        /builddir/build/BUILD/podman-4.9.0/libpod/shutdown/handler.go:48 +0x87 fp=0xc000083fe0 sp=0xc000083e58 pc=0x559bcff5a9a7
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000083fe8 sp=0xc000083fe0 pc=0x559bcf1c7f21
created by github.com/containers/podman/v4/libpod/shutdown.Start in goroutine 1
        /builddir/build/BUILD/podman-4.9.0/libpod/shutdown/handler.go:47 +0xf1

goroutine 10 [select]:
runtime.gopark(0xc000505788?, 0x2?, 0x0?, 0x0?, 0xc000505784?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000505630 sp=0xc000505610 pc=0x559bcf1940ce
runtime.selectgo(0xc000505788, 0xc000505780, 0x0?, 0x0, 0x0?, 0x1)
        /usr/lib/golang/src/runtime/select.go:327 +0x725 fp=0xc000505750 sp=0xc000505630 pc=0x559bcf1a48e5
database/sql.(*DB).connectionOpener(0xc00020a680, {0x559bd0bdd7b0, 0xc00007e2d0})
        /usr/lib/golang/src/database/sql/sql.go:1218 +0x87 fp=0xc0005057b8 sp=0xc000505750 pc=0x559bcfc0b207
database/sql.OpenDB.func1()
        /usr/lib/golang/src/database/sql/sql.go:791 +0x28 fp=0xc0005057e0 sp=0xc0005057b8 pc=0x559bcfc09628
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0005057e8 sp=0xc0005057e0 pc=0x559bcf1c7f21
created by database/sql.OpenDB in goroutine 1
        /usr/lib/golang/src/database/sql/sql.go:791 +0x165

goroutine 16 [chan receive]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc00006bf08 sp=0xc00006bee8 pc=0x559bcf1940ce
runtime.chanrecv(0xc0004bc720, 0xc00006bfc8, 0x1)
        /usr/lib/golang/src/runtime/chan.go:583 +0x3cd fp=0xc00006bf80 sp=0xc00006bf08 pc=0x559bcf15fd0d
runtime.chanrecv2(0x0?, 0x0?)
        /usr/lib/golang/src/runtime/chan.go:447 +0x12 fp=0xc00006bfa8 sp=0xc00006bf80 pc=0x559bcf15f932
github.com/containers/podman/v4/libpod.(*Runtime).startWorker.func1()
        /builddir/build/BUILD/podman-4.9.0/libpod/runtime_worker.go:9 +0x6c fp=0xc00006bfe0 sp=0xc00006bfa8 pc=0x559bd00bd2ac
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc00006bfe8 sp=0xc00006bfe0 pc=0x559bcf1c7f21
created by github.com/containers/podman/v4/libpod.(*Runtime).startWorker in goroutine 1
        /builddir/build/BUILD/podman-4.9.0/libpod/runtime_worker.go:8 +0x8e
Aborted
sh-5.2# exit
exit
PS C:\...\...>
```

### Describe the results you expected

```
PS C:\...\...> docker run -it --rm quay.io/podman/stable /bin/sh
sh-5.2# podman search --format json --list-tags mcr.microsoft.com/dotnet/sdk
[
    {
        "Name": "mcr.microsoft.com/dotnet/sdk",
        "Tags": [
            "2.1",
            "2.1-alpine",
            "2.1-alpine3.12",
            "2.1-alpine3.13",
            "2.1-bionic",
            "2.1-bionic-arm32v7",
            "2.1-focal",
            "2.1-focal-arm32v7",
            "2.1-nanoserver-1809",
            "2.1-nanoserver-1903",
            "2.1-nanoserver-1909",
            "2.1-nanoserver-2004",
            "2.1-nanoserver-2009",
            "2.1-nanoserver-20H2",
            "2.1-stretch",
            "2.1-stretch-arm32v7",
            "2.1.811",
            "2.1.811-alpine3.12",
            "2.1.811-bionic",
            "2.1.811-bionic-arm32v7",
            "2.1.811-focal",
            "2.1.811-focal-arm32v7",
            "2.1.811-nanoserver-1809",
            "2.1.811-nanoserver-1903",
            "2.1.811-nanoserver-1909"
        ]
    }
]
sh-5.2# podman search --format json --list-tags mcr.microsoft.com/dotnet/sdk
[
    {
        "Name": "mcr.microsoft.com/dotnet/sdk",
        "Tags": [
            "2.1",
            "2.1-alpine",
            "2.1-alpine3.12",
            "2.1-alpine3.13",
            "2.1-bionic",
            "2.1-bionic-arm32v7",
            "2.1-focal",
            "2.1-focal-arm32v7",
            "2.1-nanoserver-1809",
            "2.1-nanoserver-1903",
            "2.1-nanoserver-1909",
            "2.1-nanoserver-2004",
            "2.1-nanoserver-2009",
            "2.1-nanoserver-20H2",
            "2.1-stretch",
            "2.1-stretch-arm32v7",
            "2.1.811",
            "2.1.811-alpine3.12",
            "2.1.811-bionic",
            "2.1.811-bionic-arm32v7",
            "2.1.811-focal",
            "2.1.811-focal-arm32v7",
            "2.1.811-nanoserver-1809",
            "2.1.811-nanoserver-1903",
            "2.1.811-nanoserver-1909"
        ]
    }
]
sh-5.2# exit
exit
PS C:\...\...>
```

### podman info output

```yaml
I was running in a Docker container using the quay.io/podman/stable image (v4.9.0), but I was also able to reproduce the error using an alpine image and adding podman, jq, and fuse-overlayfs with no cache in the latter of course.
`podman version 4.8.3`
Windows 11 (Version 10.0.22631 Build 22631)
```

### Podman in a container

Yes

### Privileged Or Rootless

None

### Upstream Latest Release

Yes

### Additional environment details

The container is a typical docker container, so it is rootful but not privileged.

Docker Desktop was the source of the docker daemon if that matters at all. (Version 4.27.1)
It is using WSL 2.

**Comment by TuckerVaughan** [comment]
Not sure if this is related to https://github.com/containers/podman/issues/21609 or https://github.com/containers/podman/issues/21157 at all, but it seems similar to the former. I could be wrong though.

**Comment by rhatdan** [comment]
I am not able to reproduce with podman 4.9.

podman -v
podman version 4.9.0

# docker run -it --privileged --rm quay.io/podman/stable /bin/sh


I had to run privileged because docker defaults seccomp rules disbale clone syscall.

**Comment by cgruver** [comment]
Similar issue when attempting Nested Podman on OpenShift 4.14:

Reproducer:

### Create an SCC

```bash
cat << EOF | oc apply -f -
apiVersion: security.openshift.io/v1
metadata:
  name: nested-podman-scc
allowHostDirVolumePlugin: false
allowHostIPC: false
allowHostNetwork: false
allowHostPID: false
allowHostPorts: false
allowPrivilegeEscalation: true
allowPrivilegedContainer: true
allowedCapabilities:
- SETUID
- SETGID
defaultAddCapabilities: null
fsGroup:
  type: MustRunAs
groups: []
kind: SecurityContextConstraints
priority: null
readOnlyRootFilesystem: false
requiredDropCapabilities:
- KILL
- MKNOD
runAsUser:
  type: MustRunAsRange
seLinuxContext:
  type: MustRunAs
supplementalGroups:
  type: RunAsAny
users: []
volumes:
- configMap
- downwardAPI
- emptyDir
- persistentVolumeClaim
- projected
- secret
EOF
```

### Grant the SCC to a non-admin user

```bash
oc adm policy add-scc-to-user nested-podman-scc <non-admin-user>
```

### Log into the OpenShift cluster as the non-admin user

### Create a Pod

```bash
cat << EOF | oc apply -f -
apiVersion: v1
kind: Pod
metadata:
 name: nested-podman
spec:
  containers:
  - name: nested-podman
    image: quay.io/cgruver0/che/nested:latest
    args: ["tail", "-f", "/dev/null"]
    securityContext:
      privileged: true
      allowPrivilegeEscalation: true
      procMount: Unmasked
      capabilities:
        add:
        - "SETUID"
        - "SETGID"
EOF
```

### Access the shell in the pod

```bash
oc rsh nested-podman
```

### Run the following container:

```bash
podman run -d --rm --name webserver -p 8080:80 quay.io/libpod/banner
```

### Observe that the container is running and listening on port 8080:

```bash
curl http://localhost:8080
```

### Run another Podman command

```bash
podman images
```

Observe a `SIGSEGV`

## Podman Info

```bash
host:
  arch: amd64
  buildahVersion: 1.31.3
  cgroupControllers:
  - cpuset
  - cpu
  - io
  - memory
  - hugetlb
  - pids
  - rdma
  - misc
  cgroupManager: cgroupfs
  cgroupVersion: v2
  conmon:
    package: conmon-2.1.8-1.el9.x86_64
    path: /usr/bin/conmon
    version: 'conmon version 2.1.8, commit: aadb7c890ac6283eb4666d92690238e5fbdec5c7'
  cpuUtilization:
    idlePercent: 95.54
    systemPercent: 0.88
    userPercent: 3.57
  cpus: 16
  databaseBackend: boltdb
  distribution:
    distribution: '"rhel"'
    version: "9.3"
  eventLogger: file
  freeLocks: 2048
  hostname: workspace344876e4b19d4dda-7f4767dc96-c22qx
  idMappings:
    gidmap:
    - container_id: 0
      host_id: 0
      size: 1
    - container_id: 1
      host_id: 1000730001
      size: 2147483646
    uidmap:
    - container_id: 0
      host_id: 1000730000
      size: 1
    - container_id: 1
      host_id: 1000730001
      size: 2147483646
  kernel: 5.14.0-284.50.1.el9_2.x86_64
  linkmode: dynamic
  logDriver: k8s-file
  memFree: 35529629696
  memTotal: 66993119232
  networkBackend: netavark
  networkBackendInfo:
    backend: netavark
    dns:
      package: Unknown
    package: netavark-1.7.0-2.el9_3.x86_64
    path: /usr/libexec/podman/netavark
    version: netavark 1.7.0
  ociRuntime:
    name: crun
    package: crun-1.8.7-1.el9.x86_64
    path: /usr/bin/crun
    version: |-
      crun version 1.8.7
      commit: 53a9996ce82d1ee818349bdcc64797a1fa0433c4
      rundir: /run/user/1000730000/crun
      spec: 1.0.0
      +SYSTEMD +SELINUX +APPARMOR +CAP +SECCOMP +EBPF +CRIU +YAJL
  os: linux
  pasta:
    executable: ""
    package: ""
    version: ""
  remoteSocket:
    path: /run/user/1000730000/podman/podman.sock
  security:
    apparmorEnabled: false
    capabilities: CAP_CHOWN,CAP_DAC_OVERRIDE,CAP_FOWNER,CAP_FSETID,CAP_KILL,CAP_NET_BIND_SERVICE,CAP_SETFCAP,CAP_SETGID,CAP_SETPCAP,CAP_SETUID,CAP_SYS_CHROOT
    rootless: true
    seccompEnabled: true
    seccompProfilePath: /usr/share/containers/seccomp.json
    selinuxEnabled: false
  serviceIsRemote: false
  slirp4netns:
    executable: /usr/bin/slirp4netns
    package: slirp4netns-1.2.1-1.el9.x86_64
    version: |-
      slirp4netns version 1.2.1
      commit: 09e31e92fa3d2a1d3ca261adaeb012c8d75a8194
      libslirp: 4.4.0
      SLIRP_CONFIG_VERSION_MAX: 3
      libseccomp: 2.5.2
  swapFree: 0
  swapTotal: 0
  uptime: 2h 44m 59.00s (Approximately 0.08 days)
plugins:
  authorization: null
  log:
  - k8s-file
  - none
  - passthrough
  - journald
  network:
  - bridge
  - macvlan
  - ipvlan
  volume:
  - local
registries:
  search:
  - registry.access.redhat.com
  - registry.redhat.io
  - docker.io
store:
  configFile: /home/user/.config/containers/storage.conf
  containerStore:
    number: 0
    paused: 0
    running: 0
    stopped: 0
  graphDriverName: overlay
  graphOptions: {}
  graphRoot: /home/user/.local/share/containers/storage
  graphRootAllocated: 255455465472
  graphRootUsed: 49271427072
  graphStatus:
    Backing Filesystem: overlayfs
    Native Overlay Diff: "false"
    Supports d_type: "true"
    Using metacopy: "false"
  imageCopyTmpDir: /var/tmp
  imageStore:
    number: 0
  runRoot: /tmp/containers-user-1000730000/containers
  transientStore: false
  volumePath: /home/user/.local/share/containers/storage/volumes
version:
  APIVersion: 4.6.1
  Built: 1701529524
  BuiltTime: Sat Dec  2 15:05:24 2023
  GitCommit: ""
  GoVersion: go1.20.10
  Os: linux
  OsArch: linux/amd64
  Version: 4.6.1
```

Container Info: (Built from UBI9)

```bash
5.14.0-284.50.1.el9_2.x86_64 #1 SMP PREEMPT_DYNAMIC Thu Jan 18 12:28:24 EST 2024 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by cgruver** [comment]
@rhatdan I can reproduce a similar error with Podman 4.9.0.

It's the same behavior as what I posted in the previous comment.  Use `quay.io/cgruver0/che/nested-fedora:latest`

**Comment by rhatdan** [comment]
@mheon @Luap99 Didn't something like this get fixed recently?

**Comment by mheon** [comment]
Can't remember anything similar going in

**Comment by TuckerVaughan** [comment]
@rhatdan My container was not running in privileged, so I would try to find a way to run it without, unless podman requires a privileged container.

I could not find anything on this issue when I looked through the issues, but I may have missed something.

**Comment by TuckerVaughan** [comment]
@cgruver I'm not sure that issue is related as I don't see a similar error message nor did I use a file like that. I just spun up a container, ran the commands I posted, and podman failed.

**Comment by cgruver** [comment]
@TuckerVaughan  I'm seeing the same type of error that you are.  I just didn't post it.

You can ignore the `ERRO[0000] Unable to write system event: "write unixgram @3fac7->/run/systemd/journal/socket: sendmsg: no such file or directory"`  That's just because the container that I ran the test in does not have `systemd`

**Comment by TuckerVaughan** [comment]
Oh. My bad. Gotcha. 👍

**Comment by cgruver** [comment]
FWIW:  This is the type of error I am seeing:

I can generate this error simply by running `podman info` twice.  It succeeds the first time.  Then throws `SIGSEGV` every time after.

```bash
[signal SIGSEGV: segmentation violation code=0x1 addr=0x30 pc=0x56420456ad53]

goroutine 1 [running]:
panic({0x564204e99da0?, 0x564205c4e8a0?})
        /usr/lib/golang/src/runtime/panic.go:1017 +0x3ac fp=0xc000765178 sp=0xc0007650c8 pc=0x5642036a0d8c
runtime.panicmem(...)
        /usr/lib/golang/src/runtime/panic.go:261
runtime.sigpanic()
        /usr/lib/golang/src/runtime/signal_unix.go:861 +0x378 fp=0xc0007651d8 sp=0xc000765178 pc=0x5642036b8778
github.com/containers/podman/v4/libpod.(*Runtime).hostInfo(0xc0005da000)
        /builddir/build/BUILD/podman-4.9.0/libpod/info.go:129 +0x2f3 fp=0xc000765740 sp=0xc0007651d8 pc=0x56420456ad53
github.com/containers/podman/v4/libpod.(*Runtime).info(0xc0005da000)
        /builddir/build/BUILD/podman-4.9.0/libpod/info.go:40 +0x1c5 fp=0xc0007659e0 sp=0xc000765740 pc=0x56420456a3e5
github.com/containers/podman/v4/libpod.(*Runtime).Info(...)
        /builddir/build/BUILD/podman-4.9.0/libpod/runtime.go:888
github.com/containers/podman/v4/pkg/domain/infra/abi.(*ContainerEngine).Info(0xc00052a768, {0x0?, 0x0?})
        /builddir/build/BUILD/podman-4.9.0/pkg/domain/infra/abi/system.go:27 +0x30 fp=0xc000765af0 sp=0xc0007659e0 pc=0x56420468a630
github.com/containers/podman/v4/cmd/podman/system.info(0x564205c85560?, {0x564205d84ae0?, 0x0?, 0x0?})
        /builddir/build/BUILD/podman-4.9.0/cmd/podman/system/info.go:73 +0x73 fp=0xc000765ba8 sp=0xc000765af0 pc=0x564204887753
github.com/spf13/cobra.(*Command).execute(0x564205c85560, {0xc0001340b0, 0x0, 0x0})
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:983 +0xabc fp=0xc000765d48 sp=0xc000765ba8 pc=0x564203bfaebc
github.com/spf13/cobra.(*Command).ExecuteC(0x564205c6e560)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:1115 +0x3ff fp=0xc000765e20 sp=0xc000765d48 pc=0x564203bfb77f
github.com/spf13/cobra.(*Command).Execute(...)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:1039
github.com/spf13/cobra.(*Command).ExecuteContext(...)
        /builddir/build/BUILD/podman-4.9.0/vendor/github.com/spf13/cobra/command.go:1032
main.Execute()
        /builddir/build/BUILD/podman-4.9.0/cmd/podman/root.go:115 +0xb8 fp=0xc000765ea8 sp=0xc000765e20 pc=0x56420489e3f8
main.main()
        /builddir/build/BUILD/podman-4.9.0/cmd/podman/main.go:60 +0x467 fp=0xc000765f40 sp=0xc000765ea8 pc=0x56420489db27
runtime.main()
        /usr/lib/golang/src/runtime/proc.go:267 +0x2d2 fp=0xc000765fe0 sp=0xc000765f40 pc=0x5642036a3c52
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000765fe8 sp=0xc000765fe0 pc=0x5642036d7f21

goroutine 2 [force gc (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000088fa8 sp=0xc000088f88 pc=0x5642036a40ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.forcegchelper()
        /usr/lib/golang/src/runtime/proc.go:322 +0xb8 fp=0xc000088fe0 sp=0xc000088fa8 pc=0x5642036a3f38
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000088fe8 sp=0xc000088fe0 pc=0x5642036d7f21
created by runtime.init.7 in goroutine 1
        /usr/lib/golang/src/runtime/proc.go:310 +0x1a

goroutine 18 [GC sweep wait]:
runtime.gopark(0x1?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000084778 sp=0xc000084758 pc=0x5642036a40ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.bgsweep(0x0?)
        /usr/lib/golang/src/runtime/mgcsweep.go:321 +0xdf fp=0xc0000847c8 sp=0xc000084778 pc=0x56420368e53f
runtime.gcenable.func1()
        /usr/lib/golang/src/runtime/mgc.go:200 +0x25 fp=0xc0000847e0 sp=0xc0000847c8 pc=0x564203683665
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000847e8 sp=0xc0000847e0 pc=0x5642036d7f21
created by runtime.gcenable in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:200 +0x66

goroutine 19 [GC scavenge wait]:
runtime.gopark(0xc00010e000?, 0x564204c87310?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000084f70 sp=0xc000084f50 pc=0x5642036a40ce
runtime.goparkunlock(...)
        /usr/lib/golang/src/runtime/proc.go:404
runtime.(*scavengerState).park(0x564205d4db80)
        /usr/lib/golang/src/runtime/mgcscavenge.go:425 +0x49 fp=0xc000084fa0 sp=0xc000084f70 pc=0x56420368bdc9
runtime.bgscavenge(0x0?)
        /usr/lib/golang/src/runtime/mgcscavenge.go:658 +0x59 fp=0xc000084fc8 sp=0xc000084fa0 pc=0x56420368c379
runtime.gcenable.func2()
        /usr/lib/golang/src/runtime/mgc.go:201 +0x25 fp=0xc000084fe0 sp=0xc000084fc8 pc=0x564203683605
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000084fe8 sp=0xc000084fe0 pc=0x5642036d7f21
created by runtime.gcenable in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:201 +0xa5

goroutine 20 [finalizer wait]:
runtime.gopark(0x198?, 0x5642050bdc20?, 0x1?, 0x52?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000088620 sp=0xc000088600 pc=0x5642036a40ce
runtime.runfinq()
        /usr/lib/golang/src/runtime/mfinal.go:193 +0x107 fp=0xc0000887e0 sp=0xc000088620 pc=0x564203682687
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000887e8 sp=0xc0000887e0 pc=0x5642036d7f21
created by runtime.createfing in goroutine 1
        /usr/lib/golang/src/runtime/mfinal.go:163 +0x3d

goroutine 21 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000085750 sp=0xc000085730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0000857e0 sp=0xc000085750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000857e8 sp=0xc0000857e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 22 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000085f50 sp=0xc000085f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc000085fe0 sp=0xc000085f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000085fe8 sp=0xc000085fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 23 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000086750 sp=0xc000086730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0000867e0 sp=0xc000086750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000867e8 sp=0xc0000867e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 24 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000086f50 sp=0xc000086f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc000086fe0 sp=0xc000086f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000086fe8 sp=0xc000086fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 25 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000087750 sp=0xc000087730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0000877e0 sp=0xc000087750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0000877e8 sp=0xc0000877e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 26 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000087f50 sp=0xc000087f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc000087fe0 sp=0xc000087f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000087fe8 sp=0xc000087fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 27 [GC worker (idle)]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f4750 sp=0xc0004f4730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f47e0 sp=0xc0004f4750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f47e8 sp=0xc0004f47e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 28 [GC worker (idle)]:
runtime.gopark(0xa34c59453ed?, 0x1?, 0x39?, 0x48?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f4f50 sp=0xc0004f4f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f4fe0 sp=0xc0004f4f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f4fe8 sp=0xc0004f4fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 29 [GC worker (idle)]:
runtime.gopark(0xa34c599ca0e?, 0x1?, 0xbc?, 0x8a?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f5750 sp=0xc0004f5730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f57e0 sp=0xc0004f5750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f57e8 sp=0xc0004f57e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 30 [GC worker (idle)]:
runtime.gopark(0xa34c5945422?, 0x1?, 0x6f?, 0xb8?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f5f50 sp=0xc0004f5f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f5fe0 sp=0xc0004f5f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f5fe8 sp=0xc0004f5fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 31 [GC worker (idle)]:
runtime.gopark(0xa34c594509a?, 0x1?, 0x32?, 0xc7?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f6750 sp=0xc0004f6730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f67e0 sp=0xc0004f6750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f67e8 sp=0xc0004f67e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 32 [GC worker (idle)]:
runtime.gopark(0x564205d874c0?, 0x3?, 0xfc?, 0x64?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f6f50 sp=0xc0004f6f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f6fe0 sp=0xc0004f6f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f6fe8 sp=0xc0004f6fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 33 [GC worker (idle)]:
runtime.gopark(0x564205d874c0?, 0x1?, 0x6e?, 0x54?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f7750 sp=0xc0004f7730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f77e0 sp=0xc0004f7750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f77e8 sp=0xc0004f77e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 34 [GC worker (idle)]:
runtime.gopark(0xa34c599c9b4?, 0x3?, 0xf2?, 0x26?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f7f50 sp=0xc0004f7f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f7fe0 sp=0xc0004f7f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f7fe8 sp=0xc0004f7fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 50 [GC worker (idle)]:
runtime.gopark(0x564205d874c0?, 0x3?, 0x1c?, 0xf?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f0750 sp=0xc0004f0730 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f07e0 sp=0xc0004f0750 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f07e8 sp=0xc0004f07e0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 51 [GC worker (idle)]:
runtime.gopark(0xa34c599d6fc?, 0x3?, 0xc9?, 0x2?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f0f50 sp=0xc0004f0f30 pc=0x5642036a40ce
runtime.gcBgMarkWorker()
        /usr/lib/golang/src/runtime/mgc.go:1295 +0xe5 fp=0xc0004f0fe0 sp=0xc0004f0f50 pc=0x564203685225
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f0fe8 sp=0xc0004f0fe0 pc=0x5642036d7f21
created by runtime.gcBgMarkStartWorkers in goroutine 1
        /usr/lib/golang/src/runtime/mgc.go:1219 +0x1c

goroutine 3 [select, locked to thread]:
runtime.gopark(0xc0004f37a8?, 0x2?, 0x69?, 0x43?, 0xc0004f37a4?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc0004f3638 sp=0xc0004f3618 pc=0x5642036a40ce
runtime.selectgo(0xc0004f37a8, 0xc0004f37a0, 0x0?, 0x0, 0x0?, 0x1)
        /usr/lib/golang/src/runtime/select.go:327 +0x725 fp=0xc0004f3758 sp=0xc0004f3638 pc=0x5642036b48e5
runtime.ensureSigM.func1()
        /usr/lib/golang/src/runtime/signal_unix.go:1014 +0x1a5 fp=0xc0004f37e0 sp=0xc0004f3758 pc=0x5642036ce525
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f37e8 sp=0xc0004f37e0 pc=0x5642036d7f21
created by runtime.ensureSigM in goroutine 1
        /usr/lib/golang/src/runtime/signal_unix.go:997 +0xc8

goroutine 4 [syscall]:
runtime.notetsleepg(0x0?, 0x0?)
        /usr/lib/golang/src/runtime/lock_futex.go:236 +0x29 fp=0xc0004f3fa0 sp=0xc0004f3f68 pc=0x564203675429
os/signal.signal_recv()
        /usr/lib/golang/src/runtime/sigqueue.go:152 +0x29 fp=0xc0004f3fc0 sp=0xc0004f3fa0 pc=0x5642036d43c9
os/signal.loop()
        /usr/lib/golang/src/os/signal/signal_unix.go:23 +0x13 fp=0xc0004f3fe0 sp=0xc0004f3fc0 pc=0x5642037ae773
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc0004f3fe8 sp=0xc0004f3fe0 pc=0x5642036d7f21
created by os/signal.Notify.func1.1 in goroutine 1
        /usr/lib/golang/src/os/signal/signal.go:151 +0x1f

goroutine 5 [select]:
runtime.gopark(0xc0000897b0?, 0x2?, 0x0?, 0x0?, 0xc0000896ac?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000098d38 sp=0xc000098d18 pc=0x5642036a40ce
runtime.selectgo(0xc000098fb0, 0xc0000896a8, 0x0?, 0x0, 0x0?, 0x1)
        /usr/lib/golang/src/runtime/select.go:327 +0x725 fp=0xc000098e58 sp=0xc000098d38 pc=0x5642036b48e5
github.com/containers/podman/v4/libpod/shutdown.Start.func1()
        /builddir/build/BUILD/podman-4.9.0/libpod/shutdown/handler.go:48 +0x87 fp=0xc000098fe0 sp=0xc000098e58 pc=0x56420446a9a7
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000098fe8 sp=0xc000098fe0 pc=0x5642036d7f21
created by github.com/containers/podman/v4/libpod/shutdown.Start in goroutine 1
        /builddir/build/BUILD/podman-4.9.0/libpod/shutdown/handler.go:47 +0xf1

goroutine 6 [select]:
runtime.gopark(0xc000089f88?, 0x2?, 0x0?, 0x0?, 0xc000089f84?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000089e30 sp=0xc000089e10 pc=0x5642036a40ce
runtime.selectgo(0xc000089f88, 0xc000089f80, 0x0?, 0x0, 0x0?, 0x1)
        /usr/lib/golang/src/runtime/select.go:327 +0x725 fp=0xc000089f50 sp=0xc000089e30 pc=0x5642036b48e5
database/sql.(*DB).connectionOpener(0xc0003dc4e0, {0x5642050ed7b0, 0xc000532190})
        /usr/lib/golang/src/database/sql/sql.go:1218 +0x87 fp=0xc000089fb8 sp=0xc000089f50 pc=0x56420411b207
database/sql.OpenDB.func1()
        /usr/lib/golang/src/database/sql/sql.go:791 +0x28 fp=0xc000089fe0 sp=0xc000089fb8 pc=0x564204119628
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000089fe8 sp=0xc000089fe0 pc=0x5642036d7f21
created by database/sql.OpenDB in goroutine 1
        /usr/lib/golang/src/database/sql/sql.go:791 +0x165

goroutine 12 [chan receive]:
runtime.gopark(0x0?, 0x0?, 0x0?, 0x0?, 0x0?)
        /usr/lib/golang/src/runtime/proc.go:398 +0xce fp=0xc000698f08 sp=0xc000698ee8 pc=0x5642036a40ce
runtime.chanrecv(0xc0003bccc0, 0xc000698fc8, 0x1)
        /usr/lib/golang/src/runtime/chan.go:583 +0x3cd fp=0xc000698f80 sp=0xc000698f08 pc=0x56420366fd0d
runtime.chanrecv2(0x0?, 0x0?)
        /usr/lib/golang/src/runtime/chan.go:447 +0x12 fp=0xc000698fa8 sp=0xc000698f80 pc=0x56420366f932
github.com/containers/podman/v4/libpod.(*Runtime).startWorker.func1()
        /builddir/build/BUILD/podman-4.9.0/libpod/runtime_worker.go:9 +0x6c fp=0xc000698fe0 sp=0xc000698fa8 pc=0x5642045cd2ac
runtime.goexit()
        /usr/lib/golang/src/runtime/asm_amd64.s:1650 +0x1 fp=0xc000698fe8 sp=0xc000698fe0 pc=0x5642036d7f21
created by github.com/containers/podman/v4/libpod.(*Runtime).startWorker in goroutine 1
        /builddir/build/BUILD/podman-4.9.0/libpod/runtime_worker.go:8 +0x8e
zsh: IOT instruction (core dumped)  podman info
```

**Comment by Luap99** [comment]
> @mheon @Luap99 Didn't something like this get fixed recently?

yes

Duplicate of #20908

The fix is in v4.9.1, new updated container images are build once it lands in fedora stable AFAIK.

**Comment by TuckerVaughan** [comment]
@Luap99 Cool. Thanks!
